@types/ReactNative.ts

```diff
@@ -5,5 +5,6 @@ declare module "react-native" {
     export interface NativeModulesStatic {
         APIClient: NativeAPIClient;
         GenericClient: NativeGenericClient;
+        RNNCConstants: NativeConstants;
```

Comment:
I was thinking about this the other day - it's probably worth just calling this `Constants` to keep it inline with the naming above it no? `APIClient >> NativeAPIClient`, `Constants >> NativeConstants`

Reply:
Even though I agree with @migbot fo the Emm and PasteInput libraries we did not add any prefix, truth of the matter is that if you are the first to use the name why would we add a prefix, lets have others who want to use the same name to add the prefix themselves.. thoughts?

Reply:
I would change it to `NetworkConstants` though